libs/langchain-community/src/types/watsonx-types.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseLanguageModelParams } from "@langchain/core/language_models/base";
+
+export interface WatsonModelParameters {
+  decoding_method?: "sample" | "greedy";
+  max_new_tokens?: number;
+  min_new_tokens?: number;
+  stop_sequences?: string[];
+  temperature?: number;
+  top_k?: number;
+  top_p?: number;
+  repetition_penalty?: number;
+}
```

Comment:
Did a bit more digging and looks like the Random Seed could be added like this:
```suggestion
export interface WatsonModelParameters {
  decoding_method?: "sample" | "greedy";
  max_new_tokens?: number;
  min_new_tokens?: number;
  stop_sequences?: string[];
  temperature?: number;
  top_k?: number;
  top_p?: number;
  repetition_penalty?: number;
  random_seed?: number;
}
```

Reply:
Awesome, thanks! I will update it.